### PR TITLE
Refactor code quality

### PR DESCRIPTION
**Pull request by ChielBruin**
Made code improvements as suggested by Jurgen

### CODE REVIEW

**Comment by Geweldig** [comment]
Should be merged in my opinion.

File: src/main/java/nl/tudelft/contextproject/model/level/TileType.java [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
 /**
  * Enum representing the possible TileTypes in the game.
  */
 public enum TileType {
-    FLOOR(1),
-    WALL(2),
-    CORRIDOR(3);
+    FLOOR(1, 0, ColorRGBA.Green, "Textures/grasstexture.png"),
+    WALL(2, 3, ColorRGBA.Gray, "Textures/walltexture.png"),
+    CORRIDOR(3, 0, ColorRGBA.Green, "Textures/grasstexture.png");
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/model/level/TileType.java]: Indentation

**Comment by Taeir** [comment]
Apart from one indentation mistake, this looks good and can be merged.

**Comment by Taeir** [comment]
Can be merged now, (when travis is done).

**Pull request merged**